lib/inquisition/brakeman/runner.rb

```diff
@@ -1,8 +1,25 @@
+require 'brakeman'
+
 module Inquisition
   module Brakeman
     class Runner < ::Inquisition::Runner
       def call
-        puts 'Inquisition::Brakeman::Runner'
+        issues = []
+        ::Brakeman.run('./').run_checks.warnings.each do |warning|
+          issues << Issue.new(level: issue_level(warning.confidence), file: warning.file.relative,
+                              line: warning.line, message: warning.message.to_s, runner: self)
+        end
+        issues
+      end
+
+      private
+
+      def issue_level(confidence)
+        case confidence
```

Comment:
I propose to create a constant that will store matches and in the absence of a key, return the default value.